definitions/video-datazoom_video/definition.yml

```diff
@@ -0,0 +1,29 @@
+domain: VIDEO
+# Represents the video player experience across all instances of a customer's Browser Video players instrumented with New Relic's Video SDKs
+type: DATAZOOM_VIDEO
+
+synthesis:
+  rules:
+    - identifier: configuration_name
+      name: configuration_name
+
+      encodeIdentifierInGUID: false
+
+      conditions:
+        - attribute: eventType
+          value: Datazoom
+        - attribute: player_name
+          present: true
+      
+      tags:
+        player_name:
+        player_version:
```

Comment:
should these have a TTL? I can see how a player is updated to a new version and the user wants the old version to get deleted after 1d? 7d?

Reply:
1 day TTL added